在不同的机器上使用 hugegraph-tools 报错

HugeGraphServer 和 HugeGraph-tools 部署在不同的机器上，在 tools 所在的机器上调用 api，例如：

```bash
./hugegraph --graph my_graph_name --url http://my_ip:8080 task-list --status NEW
```

```java
Exception in thread "main" java.lang.RuntimeException: Construct manager failed for class 'class com.baidu.hugegraph.manager.TasksManager'
	at com.baidu.hugegraph.cmd.HugeGraphCommand.manager(HugeGraphCommand.java:272)
	at com.baidu.hugegraph.cmd.HugeGraphCommand.execute(HugeGraphCommand.java:214)
	at com.baidu.hugegraph.cmd.HugeGraphCommand.main(HugeGraphCommand.java:310)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.baidu.hugegraph.cmd.HugeGraphCommand.manager(HugeGraphCommand.java:270)
	... 2 more
Caused by: java.lang.AbstractMethodError: javax.ws.rs.core.UriBuilder.uri(Ljava/lang/String;)Ljavax/ws/rs/core/UriBuilder;
	at javax.ws.rs.core.UriBuilder.fromUri(UriBuilder.java:119)
	at org.glassfish.jersey.client.JerseyWebTarget.<init>(JerseyWebTarget.java:71)
	at org.glassfish.jersey.client.JerseyClient.target(JerseyClient.java:290)
	at org.glassfish.jersey.client.JerseyClient.target(JerseyClient.java:76)
	at com.baidu.hugegraph.rest.RestClient.<init>(RestClient.java:85)
	at com.baidu.hugegraph.rest.RestClient.<init>(RestClient.java:62)
	at com.baidu.hugegraph.client.RestClient.<init>(RestClient.java:46)
	at com.baidu.hugegraph.driver.HugeClient.<init>(HugeClient.java:65)
	at com.baidu.hugegraph.driver.HugeClient.<init>(HugeClient.java:58)
	at com.baidu.hugegraph.base.ToolClient.<init>(ToolClient.java:40)
	at com.baidu.hugegraph.base.ToolManager.<init>(ToolManager.java:36)
	at com.baidu.hugegraph.manager.TasksManager.<init>(TasksManager.java:48)
	... 7 more
```

然而，在这台机器上使用 `curl` 可以访问 server 端的 API。在同样的机器上不指定 `--url` 参数（即使用 http://localhost:8080 默认值）运行上述命令也没有问题。

--- a/pom.xml
+++ b/pom.xml
@@ -67,17 +67,53 @@
                     <groupId>javax.ws.rs</groupId>
                     <artifactId>jsr311-api</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-server</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-core</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-servlet</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-json</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-hdfs</artifactId>
             <version>${hadoop-version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-server</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-core</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
             <version>${hadoop-version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-client</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-core</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>com.google.guava</groupId>

